src/Symfony/Component/Validator/Resources/translations/validators.it.xlf

```diff
@@ -426,6 +426,10 @@
                 <source>Using hidden overlay characters is not allowed.</source>
                 <target>Non è consentito utilizzare caratteri sovrapposti nascosti.</target>
             </trans-unit>
+            <trans-unit id="110">
+                <source>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</source>
+                <target>L'estensione del file non è valida ({{ extension }}). L'estensioni consentite sono {{ extensions }}.</target>
```

Comment:
What is the difference between this and

```
Estensioni consentite sono {{ extensions }}.
```

Reply:
It is the same, L'estensioni i would say is the school way, and Estensioni without the prefix is more like a shorter version, street like... Maybe not a great comparison but something like what's up and wassup. I exagerated a bit with the analogy just to prove the point.

Reply:
And beside that since I'm bilingual, mother from italy, father from Serbia. There is the thing that when i say it out loud it just sounds better to me, like more serious. Anyway both are correct.

Reply:
I see. So "L'estensioni" is more formal. 

Thank you for the explanation.

Reply:
Another example would be: Estensioni or L'estensioni consentite sono i would translate that: L'Estensioni valide sono, but didn't change it since all the others were in the same manner. consentite means allowed, valide means valid, even though it is more logical to use consentite, it just sounds right to say valide instead. Maybe i got too much in depth, sorry. :D

Reply:
You can't write "l'estensioni" in Italian. The only possible correct form is "le estensioni". The elision of the article is only used for the singular

Reply:
Exactly, you said exatly like I meant, I just didn't remember the word formal... :) It is a native language to me. It is properly translated, I didn't use any translators, no need to worry :) My mother is the court translator for italian, I can check with her also, but I know it is right. :D